openhands/resolver/README.md

```diff
@@ -1,191 +1,141 @@
+  - "contents"
+  - "issues"
+  - "pull requests"
+  - "workflows"
 
-   Note: If you're working with an organizational repository, you may need to configure the organization's personal access token policy first. See [Setting a personal access token policy for your organization](https://docs.github.com/en/organizations/managing-programmatic-access-to-your-organization/setting-a-personal-access-token-policy-for-your-organization) for details.
+- Create an LLM API key (e,g [Claude API](https://www.anthropic.com/api))
 
-2. Create an API key for the [Claude API](https://www.anthropic.com/api) (recommended) or another supported LLM service
+### Installation
 
-3. Copy `examples/openhands-resolver.yml` to your repository's `.github/workflows/` directory
+1. Copy `examples/openhands-resolver.yml` to your repository's `.github/workflows/` directory
 
-4. Configure repository permissions:
-    - Go to `Settings -> Actions -> General -> Workflow permissions`
-    - Select "Read and write permissions"
-    - Enable "Allow Github Actions to create and approve pull requests"
+2. Configure repository permissions:
 
-    Note: If the "Read and write permissions" option is greyed out:
-    - First check if permissions need to be set at the organization level
-    - If still greyed out at the organization level, permissions need to be set in the [Enterprise policy settings](https://docs.github.com/en/enterprise-cloud@latest/admin/enforcing-policies/enforcing-policies-for-your-enterprise/enforcing-policies-for-github-actions-in-your-enterprise#enforcing-a-policy-for-workflow-permissions-in-your-enterprise)
+   - Go to `Settings -> Actions -> General -> Workflow permissions`
+   - Select **Read and write permissions**
+   - Enable **Allow Github Actions to create and approve pull requests**
+
+   > If "Read and write permissions" is greyed out:
+   >
+   > - Check organization settings first
+   > - Otherwise, permissions might need to be set in [Enterprise policy settings](https://docs.github.com/en/enterprise-cloud@latest/admin/enforcing-policies/enforcing-policies-for-your-enterprise/enforcing-policies-for-github-actions-in-your-enterprise#enforcing-a-policy-for-workflow-permissions-in-your-enterprise)
+
+3. Set up [GitHub secrets](https://docs.github.com/en/actions/security-for-github-actions/security-guides/using-secrets-in-github-actions):
 
-5. Set up [GitHub secrets](https://docs.github.com/en/actions/security-for-github-actions/security-guides/using-secrets-in-github-actions):
    - Required:
-    - `LLM_API_KEY`: Your LLM API key
+     - `LLM_API_KEY`: Your LLM API key
    - Optional:
      - `PAT_USERNAME`: GitHub username for the personal access token
      - `PAT_TOKEN`: The personal access token
      - `LLM_BASE_URL`: Base URL for LLM API (only if using a proxy)
+     - [See how to customize more configurations](https://docs.all-hands.dev/modules/usage/how-to/github-action#custom-configurations)
+
+## 2. Setting up GitLab (CI Runner)
```

Comment:
@wtiger9218 if you could take a look at this section that would be super helpful. I've split the section into `Prerequisites` and `Installation` (you can reference the structure for github actions above if its helpful)